scripts/saveCanvas.js

```diff
@@ -0,0 +1,24 @@
+/**
+  Fetch the elements within the canvas then, using _serializeCanvas,
+  create a string represneting that data. Store the resulting string
+  at localStorage.set(storageName).
+
+   @param {string} storageName - a key for a value in localStorage
```

Comment:
Just looked up what these @param comments mean, seems more useful once you have a lot of code to read through.